Tests/secrets_white_list.json

```diff
@@ -2674,7 +2674,10 @@
     "vulnInfoList",
     "NAS-Identifier=de-wilm-251littl-idf3b-s2",
     "NAS-Port-Id=GigabitEthernet2/0/05",
-    "--memory=1g##--memory-swap=-1##--pids-limit=256##--ulimit=nofile=1024:8192"
+    "--memory=1g##--memory-swap=-1##--pids-limit=256##--ulimit=nofile=1024:8192",
+    "@pytest.mark.parametrize",
+    "TestGetSuccessfulAndFailedPacks",
+    "user_pack_metadata.json"
```

Comment:
why were these added?

Reply:
I ran `demisto-sdk secrets` and I got those strings as secrets